app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def not_found
```

Comment:
Dołączać do controllerów co prawda można, ale raczej chodzi o późniejsze użycie metod takiego helpera w widokach lub do renderowania widoku w metodzie controllera (np. metoda konwertująca tekst) - a tutaj było to rzucanie wyjątku.

http://api.rubyonrails.org/classes/ActionController/Helpers.html